Tiltfile

```diff
@@ -0,0 +1,14 @@
+# Deploy: tell Tilt what YAML to deploy
+k8s_yaml('deploy/kubernetes-manifests.yaml')
+
+custom_build(
+  'deislabs/smi-adapter-istio',
+  'operator-sdk build $EXPECTED_REF',
+  ['./pkg', './cmd'],
+  tag='latest',
+  live_update=[
+    sync('pkg', '/go/src/github.com/deislabs/smi-adapter-istio/pkg'),
```

Comment:
Does this path get autodetected in the case /go/src is NOT your $GOPATH can you make it use the value of $GOPATH instead?

Reply:
This line says copy the `pkg` directory relative to the Tilefile (in your local dev environment) to the path `/go/src/github.com/deislabs/smi-adapter-istio/pkg` inside of your container filesystem. That path doesn't need to change bc it is fixed.